GenderPayGap.WebJob/Functions/FrequentTimerTriggeredWebJobs/Functions.SendReminderEmails.cs

```diff
@@ -77,22 +77,24 @@ private void SendReminderEmailsForSectorType(SectorTypes sector, string runId, D
 
         private void CheckUserAndSendReminderEmailsForSectorType(User user, SectorTypes sector)
         {
-            List<Organisation> inScopeOrganisationsForUserAndSectorTypeThatStillNeedToReport = user.UserOrganisations
+            List<Organisation> inScopeActiveOrganisationsForUserAndSectorTypeThatStillNeedToReport = user.UserOrganisations
                 .Where(uo => uo.PINConfirmedDate != null)
                 .Select(uo => uo.Organisation)
+                .Where(o => o.Status == OrganisationStatuses.Active)
                 .Where(o => o.SectorType == sector)
                 .Where(
                     o =>
+                        o.LatestScope.Status == ScopeRowStatuses.Active && (
```

Comment:
This was an extra after-thought but feels like we should be a bit more careful

Reply:
Ooh yes, good spot